packages/fast-documentation/src/hero/hero.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import manageJss, { ComponentStyles } from "@microsoft/fast-jss-manager-react";
+import {
+    Background,
+    CallToAction,
+    CallToActionAppearance,
+    Divider,
+    DividerRoles,
+    Heading,
+    HeadingClassNameContract,
+    HeadingSize,
+    HeadingTag
+} from "@microsoft/fast-components-react-msft";
+import { DesignSystem } from "src/design-system";
+import { applyScaledTypeRamp, neutralForegroundRest } from "@microsoft/fast-components-styles-msft";
+import { breakpoints } from "../../app/data/default-vars";
+
+interface HeroClassNameContract {
+    hero: string;
+    headingWrapper: string;
+    imageZ0: string;
+}
+
+const heroStyles: ComponentStyles<HeroClassNameContract, undefined> = {
+    hero: {
+        height: "100vh",
+        position: "relative",
+        overflow: "hidden",
+        minHeight: "650px",
+        title: {
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+            justifyContent: "center",
+            height: "100%"
+        }
+    },
+    headingWrapper: {
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        textAlign: "center",
+        height: "100vh",
+        minHeight: "650px",
+        position: "relative",
+        zIndex: "5",
+        padding: "0 5%"
+    },
+    imageZ0: {
+        position: "absolute",
+        top: "0",
+        left: "0",
+        zIndex: "0",
+        minWidth: "100%",
+        height: "100vh",
+        overflow: "hidden"
+    }
+};
+
+const headingStyles: ComponentStyles<HeadingClassNameContract, DesignSystem> = {
+    heading__3: {
+        position: "relative",
+        [`@media only screen and (max-width: ${breakpoints.vp2Max})`]: {
+            ...applyScaledTypeRamp("t5")
+        }
+    }
+};
+
+const designSystemMergingFunction: (config: DesignSystem) => DesignSystem = (config: DesignSystem): DesignSystem =>
+    Object.assign({}, config, { backgroundColor: "#C22551", density: 2 });
+
+const BaseHero: React.FC<{
+    managedClasses: HeroClassNameContract;
+    foregroundPosition?: { top?: string; right?: string; left?: string; bottom?: string };
+    srcBackground?: (className: string) => React.ReactNode;
+    text: string;
+    backgroundColor?: string;
+    tall?: boolean;
+    destination: string;
+    callToAction: string;
```

Comment:
Could we just create this as an interface so we don't need to type props below as `any`?

Reply:
Would prefer to make structure updates during the next round of marketing design and content edits.